runtime/altair/src/lib.rs

```diff
@@ -1160,6 +1160,7 @@ impl pallet_interest_accrual::Config for Runtime {
 	type Balance = Balance;
 	type Event = Event;
 	type InterestRate = Rate;
+	type MaxRateCount = MaxActiveLoansPerPool;
```

Comment:
Why would this be equivalent?

Reply:
The current benchmarking code requires `MaxRateCount` to be greater or equal than `MaxActiveLoansPerPool` (it creates a unique rate for every loan).

The right values for MaxRateCount and MaxActiveLoansPerPool need to be udpated based on benchmarking, but I haven't done production benchmarking on this version of the code - I was gonna wait for Part 2 for that. I can add a TODO comment and a github ticket to adjust this based on benchmarking results.

Reply:
In other words, it's an expedient temporary value

Reply:
Makes sense. I think adding a TODO comment would be great 👍